hr_timesheet_current/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Victor Champonnois
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Hr Timesheet Current',
+    'description': """
+        Adds "my current timsheet" menu and open it directly when selecting the Timesheet menu""",
```

Comment:
```suggestion
    'summary': """
        Allow to access the current timesheet sheet directly from the menu""",
```
`summary` is what is displayed in the apps list.